### DISCUSSION TITLE
MySQL数据库安装 | 云泽的博客

### DISCUSSION THREAD

**Discussion by giscus** [codeyunze/blog#5] [General]
# MySQL数据库安装 | 云泽的博客

包含Linux和Windows两个版本MySQL数据库的详细安装步骤。

https://3xj.club/blog/database/mysql/MySQL%E6%95%B0%E6%8D%AE%E5%BA%93%E5%AE%89%E8%A3%85.html

**Comment by codeyunze** [comment]
很详细
